ios/bottom-tabs/accessory/RNSBottomAccessoryHelper.h

```diff
@@ -0,0 +1,72 @@
+#import <RNSDefines.h>
+
+#define BOTTOM_ACCESSORY_AVAILABLE RNS_IPHONE_OS_VERSION_AVAILABLE(26_0) && !TARGET_OS_TV && !TARGET_OS_VISION
+
+#if BOTTOM_ACCESSORY_AVAILABLE
+
+#import <UIKit/UIKit.h>
+#import "RNSBottomTabsAccessoryComponentView.h"
+#import "RNSBottomTabsAccessoryContentComponentView.h"
+#import "RNSEnums.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * @class RNSBottomAccessoryHelper
+ * @brief Class responsible for managing accessory size and environment changes for
+ * RNSBottomTabsAccessoryComponentView.
+ */
+API_AVAILABLE(ios(26.0))
+@interface RNSBottomAccessoryHelper : NSObject
+
+- (instancetype)initWithBottomAccessoryView:(RNSBottomTabsAccessoryComponentView *)bottomAccessoryView;
+
+/**
+ * Registers KVO for frames of UIKit's bottom accessory wrapper view.
+ * It must be called after `RNSBottomTabsAccessoryComponentView` or its ancestor is set as `bottomAccessory` on
+ * `RNSTabBarController`.
+ */
+- (void)registerForAccessoryFrameChanges;
+/**
+ * Invalidates observers, display link (if it is used); resets internal properties.
+ */
+- (void)invalidate;
+
+@end
+
+#pragma mark - Content view switching workaround
+
+#if defined(__cplusplus) && REACT_NATIVE_VERSION_MINOR >= 82
```

Comment:
Applying `defined(__cplusplus)` generally excludes these symbols from being visible from Swift.

I'm not decided what we should do here. Most likely we can stick to the ObjectiveC for now & this won't be a problem for a long time... I'm willing to 🙈 and approve this, but we need to create ticket to refactor this condition. 

Maybe, in the future, we should use a technique similar to what's already done in [`RNSDefines`](https://github.com/software-mansion/react-native-screens/blob/b2a8e0121dc936d5f6d080348a2fd6009d021ead/ios/utils/RNSDefines.h#L13-L23), instead of using `cxx/ReactNativeVersion`? That's not really a scalable solution, though. Let's create ticket for this & think there.